accounts/models.py

```diff
@@ -3,19 +3,37 @@
 from django.contrib.auth import get_user_model
 from django.contrib.auth.models import AbstractUser
 from django.db import models
+from phonenumber_field.modelfields import PhoneNumberField
 
 
 class User(AbstractUser):
     # implicit username, email, first_name, and last_name fields
     # from AbstractUser that contains the user's PennKey
     pennid = models.IntegerField(primary_key=True)
     uuid = models.UUIDField(unique=True, default=uuid.uuid4, editable=False)
+    preferred_name = models.CharField(max_length=100, blank=True)
```

Comment:
I think we should keep the max length as 255 just because everything else has that same length